src/marqo/core/index_management/index_management.py

```diff
@@ -161,6 +167,14 @@ def batch_create_indexes(self, marqo_index_requests: List[MarqoIndexRequest]) ->
                 if self.index_exists(index.name):
                     raise IndexExistsError(f"Index {index.name} already exists")
 
+
+        # Populate the prefix fields if they are None
+        for index in marqo_index_requests:
+            if index.model.text_query_prefix is None:
+                index.model.text_query_prefix = index.model._get_default_prefix("text_query_prefix")
+            if index.model.text_chunk_prefix is None:
+                index.model.text_chunk_prefix = index.model._get_default_prefix("text_chunk_prefix")
```

Comment:
Two issues here. 1. you're not covering batch create index. 2. you're calling a private method of another class here (your IDE will be complaining too)

Reply:
Maybe just add a public `get_default_text_query_prefix` and `get_default_text_chunk_prefix`. Add an index management test to make sure this works as expected

Reply:
Added